src/scripts.js

```diff
@@ -47,17 +49,24 @@ function getRandomIndex(array) {
 }
 
 function displayHomePage() {
-  displayAllRecipes();
+  displayRecipes(recipeRepository.recipeData)
   searchBar.value = '';
   // filterTags.value = '';
   hide([selectedRecipeView, homeButton]);
-  show([mainView, recipeSection, searchBar, searchRecipesButton]);
+  show([mainView, recipeSection, searchBar, searchRecipesButton, favoritePageButton]);
 }
 
 function displayFavorites() {
-  hide([favoritePageButton, mainView, recipeSection]);
-  show([homeButton]);
-  //need other functions to be built out in order to finish this
+  hide([favoritePageButton]);
+  show([homeButton, filterSection, recipeSection, mainView]);
+  displayRecipes(currentUser.favorites);
+  recipeCards = document.querySelectorAll('.recipe-card-js');
+  recipeCards.forEach(card => {
```

Comment:
Glad we landed here after taking some time to realize we were over-complicating things.